bundles/org.openhab.binding.elgatokeylight/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="elgatokeylight" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Elgato Key Light Binding</name>
+	<description>This is the binding for Elgato Key Light.</description>
```

Comment:
Can you be a bit more verbose, that anybody who doesn't heard about Elgato Key Light, gets a clue what this is about?